src/Message/ApplePayAuthorizeResponse.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+namespace Omnipay\Vindicia\Message;
+
+class ApplePayAuthorizeResponse extends Response
+{
+    /**
+     * Is the response successful?
+     * Throws an exception if there's no code.
+     *
+     * @return boolean
+     * @throws \Omnipay\Common\Exception\InvalidResponseException
+     */
+    public function isSuccessful()
+    {
+        $statusCode = $this->getStatusCode();
+        return ($statusCode >= 200 && $statusCode < 300) || $statusCode == 304;
```

Comment:
what does a 304 mean? and can they really return that many different 200s?

Reply:
I'll just leave it at 200 for now. If any other codes pop up we can update it.